src/node/node_cfg.ml

```
@@ -298,7 +298,8 @@ module Node_cfg = struct
         let cfg =  ClientCfg.from_file "nursery" filename in
         Some (n_cluster_id, cfg)
       end
-    with ex ->
+    with exn ->
+      let () = ignore exn in 
```

Hmm, catch all handlers are in general not a good idea...
I don't know exactly which exceptions are expected here... maybe Not_found?
You'll probably want to check the Ini implementation details.